manifests/params.pp

```diff
@@ -29,9 +29,18 @@
   if $::operatingsystem == 'Ubuntu' {
     if versioncmp($::lsbdistrelease, '8.04') < 1 {
       $init_style = 'debian'
-    } else {
+    } elsif versioncmp($::operatingsystemrelease, '15.04') < 0 {
       $init_style = 'upstart'
     }
+    else {
```

Comment:
Mainly because systemd is now being transitioned to and Ubuntu was annoying about it.  I figured it might be helpful to everyone.  It may or may not be good to have in a module rather than in a private wrapper module which we would load on the side to do that suggested change.  At least the fact may likely help someone :) whether in this module or not.   And yes we have a facts module for our own Uni. that contains it as well.